lib/bench/Bench/Pos/Diffusion/BlockDownload.hs

```diff
@@ -0,0 +1,325 @@
+-- A program which runs two diffusion layers over a TCP transport, and has
```

Comment:
I wrote this file, someone else should review it.

Reply:
LGTM.

Reply:
Hehe, very impartial ;) Good enough for me